ui/src/main/js/page/channel/jira/common/JiraFieldMapDistributionTable.js

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import Table from 'common/component/table/Table';
+import JiraFieldMapEditCell from 'page/channel/jira/common/JiraFieldMapEditCell';
+import JiraFieldMapTableActions from 'page/channel/jira/common/JiraFieldMapTableActions';
+
+const emptyTableConfig = {
+    message: 'There are no records to display for this table.  Please add a Jira Cloud field mapping to use this table.'
+};
+
+function JiraFieldMapDistributionTable({ cloudTableData, onFieldMappingUpdate }) {
```

Comment:
`JiraFieldMapDistributionTable` is used in both `jira/cloud/JiraCloudDistributionConfiguration` and `jira/server/JiraServerDistributionConfiguration`.  Because of that, I opted to create a `jira/common/` directory and placed `JiraFieldMapDistributionTable` and a few other commonly shared files for jira in there.  This comment did have me review the code and made me update some of the props/text to be more dynamic. Thanks!